src/utils/resolve-to-type-wrapper.util.mts

```diff
@@ -12,15 +12,6 @@ import { isTypeWrapper } from './is-type-wrapper.util.mjs';
  * @param target the class definition of the target of the decorator
  */
 export function resolveToTypeWrapper(typeOrIdentifier: AnyInjectIdentifier): TypeWrapper {
-  /**
```

Comment:
Perhaps these comments should be re-added (but actually relevant in the context of the API changes)?  For context, these haven't changed since typestack -- they would probably just confuse readers more than help them.